src/tabs/src/components/FolderTab.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import Tab from '@material-ui/core/Tab';
+import styled from 'styled-components';
+import { withStyles } from '@material-ui/core/styles';
+
+import { PURPLE } from '../../../colors';
+
+// adjust tab content to take size of bold font
+const ContentSpan = styled.span`
+  &:before {
+    display: block;
+    content: ${(props) => (props.content ? `"${props.content}"` : '')};
+    font-weight: 600;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+`;
+
+const FolderTab = withStyles((theme) => ({
+  root: {
+    textTransform: 'none',
+    minWidth: 72,
+    fontWeight: theme.typography.fontWeightRegular,
+    marginRight: theme.spacing(1),
+    backgroundColor: theme.palette.action.hover,
+    minHeight: 32,
+    borderRadius: '3px 3px 0px 0px',
+    '&:hover': {
+      boxShadow: theme.shadows[5],
+    },
```

Comment:
I'm not a fan of the box shadow effect in the gif. I like how radio and checkbox buttons are styled. How do you feel about following that pattern?


https://user-images.githubusercontent.com/440299/126225966-a8f63d37-4724-4995-a735-230b56d5f6c5.mov